moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -920,10 +920,8 @@ export const ProjectActivityLogCreateDescriptions = {
   },
   moped_proj_partners: {
     label: (record, userList) => {
-      return (
-        fieldFormat(record.record_data.event.data.new.partner_name, true) +
```

Comment:
i wanted to change this to display ``proj_partner_id`` instead of ``partner_name`` in the activity log but there is no way locally to test whether that would work, so we went with this safe option that should not break anything. i don't know that this is relevant because it will probably be overhauled with the activity log overhaul, but i just wanted to point it out! this was a learning experience for me better understanding how the activity log works. @chiaberry

Reply:
@chiaberry what do you think about this? since we don't have a way to test, it might be best to wait to merge until we migrate this table to a REST connector?

Reply:
I think we can merge this now, if we are okay with just seeing the generic "New partner added" in the meantime

Reply:
i went ahead and deleted this whole entry since i don't think this is necessary anymore